.cirrus.yml

```diff
@@ -5,10 +5,7 @@ task:
   use_compute_credits: $CIRRUS_BRANCH == 'master'
   node_modules_cache:
     folder: node_modules
-    fingerprint_script: cat package-lock.json
-    reupload_on_changes: false
-    populate_script: npm ci
-  bootstrap_script: npm run bootstrap
+  populate_script: npm i
```

Comment:
The reason I did this is to A. save time where possible for the build and B. so CRA always gets patched.  If no updates are required, it doesn't actually add any time to the build since npm will just say up-to-date and then do the patching.